package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "BarcodeScannerForPhoneGap",
+  "version": "3.0.1",
```

Comment:
Check the current version of the Appstore and update accordingly...It's a new feature so, i guess it should be 3.1 now

Reply:
Version Bumped up